examples/skr/aks/README.md

```diff
@@ -0,0 +1,228 @@
+- [Export Environment Variables](#export-environment-variables)
+- [Secret Provisioning Workflow Step By Step](#secret-provisioning-workflow)
+  - [1. Create a mHSM Instance](#1-create-a-mhsm-instance)
+  - [2. Obtain an Attestation Endpoint](#2-obtain-an-attestation-endpoint)
+  - [3. Generate a Key Pair](#3-generate-a-key-pair)
+  - [4. Generate a Wrapped Secret](#4-generate-a-wrapped-secret)
+  - [5. Create a Federated Credential Identity](#5-create-a-federated-credential-identity)
+  - [6. Build Required Container Images](#6-build-required-container-images)
+  - [7. Running the SKR Container in a TEE](#7-running-the-skr-container-in-a-tee)
+  - [8. Unwrap the Secret Using grpcurl](#8-unwrap-the-secret-using-grpcurl)
+
+## Introduction
+
+This guide provides instructions on how to perform secret provisioning in AKS using the SKR container.
+By following these instructions, users can ensure that their secrets are securely stored and can be used by their applications.
+
+## Export Environment Variables
+
+```bash
+export SKR_IMAGE=<skr-image-name>
+export EXAMPLE_UNWRAP_IMAGE=<example-unwrap-image-name>
+export RESOURCE_GROUP=<resource-group-name>
+export USER_ASSIGNED_IDENTITY_NAME=<identity-name>
+export LOCATION=<location>
+export SUBSCRIPTION="$(az account show --query id --output tsv)"
+export SERVICE_ACCOUNT_NAME=<service-account-name>
+export SERVICE_ACCOUNT_NAMESPACE="default"
+export FEDERATED_CREDENTIAL_IDENTITY_NAME=<federated-credential-identity-name>
+```
+
+## Secret Provisioning Workflow
+
+### 1. Create a mHSM Instance
+
+Users must have a functioning mHSM to store the private key and its associated release policy.
+The release policy describes the conditions the key release request must meet in order for the mhsm to release the key.
+
+To set up a mHSM instance, follow the instructions [here with Azure CLI](https://learn.microsoft.com/en-us/azure/key-vault/managed-hsm/quick-create-cli),
+or through the [Azure portal](https://ms.portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.KeyVault%2FmanagedHSMs).
+You can follow the installation instructions for Azure CLI [here](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli).
+
+### 2. Obtain an Attestation Endpoint
+
+Users must have a working attestation endpoint to interact with the mHSM.
+
+If you don't already have a valid attestation endpoint, create a [Microsoft Azure Attestation](https://learn.microsoft.com/en-us/azure/attestation/overview) endpoint to author the attestation token and run the following command to get the endpoint value:
+
+```shell
+az attestation show --name "<ATTESTATION PROVIDER NAME>" --resource-group "<RESOURCE GROUP>"
+```
+
+### 3. Generate a Key Pair
+
+This is a one time effort.
+Once the key is created, it can be used to protect as many secrets as desired.
+
+The `setup-key-mhsm.sh` script creates a private/public key pair in the keyvault and a key release policy with the file name `keyname-release-policy.json`.
```

Comment:
This script was updated to support creating both akv and mhsm. it's now called setup-key.sh

Reply:
where is it?

Reply:
nvm. That script is in the demos repo.